pkg/controller/datavolume/util.go

```diff
@@ -351,6 +351,55 @@ func updateDataVolumeUseCDIPopulator(syncState *dvSyncState) {
 	cc.AddAnnotation(syncState.dvMutated, cc.AnnUsePopulator, strconv.FormatBool(syncState.usePopulator))
 }
 
+func updateDataVolumeDefaultInstancetypeLabels(client client.Client, syncState *dvSyncState) error {
+	// Skip looking anything up if any default instance type labels are already present
+	dv := syncState.dvMutated
+	for _, defaultInstanceTypeLabel := range cc.DefaultInstanceTypeLabels {
+		if _, ok := dv.Labels[defaultInstanceTypeLabel]; ok {
+			return nil
+		}
+	}
+	if dv.Spec.Source != nil && dv.Spec.Source.PVC != nil {
```

Comment:
what about `dv.Spec.Source.Snapshot`?

Reply:
Eventually `DataImportCrons` will create "golden snapshots" instead of "golden pvcs" for provisioners in which that is optimal

Reply:
ACK thanks I thought you might say that, definitely something to follow up with in that case!